stellar-dotnet-sdk/responses/TransactionResponse.cs

```diff
@@ -57,7 +57,7 @@ public class TransactionResponse : Response, IPagingToken
         public List<string> Signatures { get; private set; }
 
         [JsonProperty(PropertyName = "fee_bump_transaction")]
-        public FeeBumpTransaction FeeBumpTransaction { get; set; }
+        public FeeBumpTransaction FeeBumpTx { get; set; }

```

Comment:
I don't think this rename should occur as it's a breaking change. Should probably leave it as it was...